README.md

```diff
@@ -79,10 +79,31 @@ Our staging and production environments are hosted by AWS. We would deploy to pr
 
 ## Testing
 
-To run tests:
+### Run the tests
 ```sh
 $ make test
 ```
+### Agreed Testing Approach
+- Use nUnit and Moq
+- Always follow a TDD approach
+- Tests should be independent of each other
+- Gateway tests should test against a real database
```

Comment:
Should we mention here that the real database would be a real instance of the database set up in docker for testing (not the actual instance in AWS)?

Reply:
I feel saying "against a real database" might imply running the gateway tests against the actual Postgres DB in AWS

Reply:
Yeah good point